app/cypress/integration/cif/project-revision/index.spec.js

```diff
@@ -111,6 +111,24 @@ describe("the new project page", () => {
     cy.happoAndAxe("Project summary Form", "empty", "main", true);
   });
 
+  it("renders the IA specific project forms", () => {
+    cy.mockLogin("cif_admin");
+    cy.visit("/cif/projects");
+    cy.get("button").contains("Add a Project").click();
+
+    // NEW
+    cy.url().should("include", "/new");
+    cy.findByLabelText(/Funding Stream$/i).should("be.visible");
+    cy.happoAndAxe("IA Project New Form", "unfilled", "main");
+    cy.fillAndCheckNewProjectForm("Innovation Accelerator", "2021");
+    cy.findByRole("button", { name: /^confirm/i }).click();
+
+    // PROJECT SUMMARY REPORT
+    cy.findByText(/Project summary report/i).click();
+    cy.findByText(/Add Project Summary Report/i).click();
+    cy.contains(/Project Summary Report Form Placeholder/i);
```

Comment:
I've added the e2e-related dev checklist item to the project summary report form card: https://github.com/bcgov/cas-cif/issues/1061